docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
+. [[ref:51]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.2. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:52]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.15. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:53]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.16. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:54]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.22. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:55]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.17. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:56]] Costetchi, E. (2020) eProcurement UML conceptual model conventions, sec 4.3. Available at: https://github.com/OP-TED/model2owl/blob/master/doc/uml-conventions/uml-conventions.pdf.
+. [[ref:57]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.3.4. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:58]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.3.1. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:59]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.3.5. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:60]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.3.3. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:61]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.3.2. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:62]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.2.5. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:63]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.1.11. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:64]] Buitelaar.P et al. (2011). Ontology Lexicalisation: The lemon Perspective.
 . [[ref:65]] Vann
-. [[ref:66]] https://www.w3.org/TR/shacl/
-. [[ref:67]] https://shex.io/
+. [[ref:66]] Knublauch, H. and Kontokostas, D. (eds) (2017) Shapes constraint language (SHACL), W3C. W3C. Available at: https://www.w3.org/TR/shacl/.
+. [[ref:67]]Shape expressions(ShEx). Available at: https://shex.io/ (Accessed: January 2, 2023).
```

Comment:
There is a standard for citing specifications.

Perhaps it is better to create a `*.bibtex` database and then use it to format citations. 
feel free to use  [https://github.com/OP-TED/model2owl/blob/master/doc/references/references.bib](https://github.com/OP-TED/model2owl/blob/master/doc/references/references.bib)

Reply:
I usually use bibtex. but here some references exist and for all references that need to be completed I uses (CITE THIS FOR ME) (HARVARD STYLE)

Reply:
I converted to one format